### PR TITLE
Components setup

### PR DESCRIPTION
**Pull request by inakiesk**
This branch is dedicated to developing the basic structure of the calculator in the Math magicians project, the features are the following: 

- Created components directory with Calculator.js file
- Created a react component in the Calculator.js file that displays the basic UI of the calculator.
- Used a class-based component.
- Calculator is imported successfully.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <div className="calcRow">
+            <CalculatorBtn value="0" styling="btn0" />
+            <CalculatorBtn value="." styling="calcBtn" />
+            <CalculatorBtn value="=" styling="calcBtn operator" />
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (mckent05) [inline src/components/Calculator.js]: - According to the project requirements, this component should be a functional component, please fix this 🙏

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,13 @@
...
-    </div>
-  );
+export default class App extends React.PureComponent {
+  render() {
+    return (
+      <div>
+        <Calculator />
+      </div>
+    );
+  }
```
Comment (mckent05) [inline src/App.js]: - According to the project requirements, please ensure that this component is a functional component, please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @inakiesk, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @inakiesk 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**